build.gradle

```diff
@@ -32,11 +32,7 @@ dockerCompose {
 }
 
 dependencies {
-    compile group: 'org.springframework', name: 'spring-context', version: '5.2.5.RELEASE'
-    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-
+    alfrescoProvided("org.alfresco:alfresco-repository:8.157")
```

Comment:
please realize our primary use case is currently alfresco 5.2, which has an incompatible quartz version :-x

Reply:
Do we want 2 separate subprojects for alfresco 5 and 6?

Reply:
And probably share a the core action ... so 3 :-)